### PR TITLE
[ACR-1129] Add new content features (Update system.md)

### PR DESCRIPTION
**Pull request by haejeongkim**
Add Content Features (ACR-1129)

### Change Description ###

Feature added
http://tizen.org/feature/content.scanning.others (new), http://tizen.org/feature/content.filter.pinyin (new)

### API Changes ###

 - ACR: ACR-1129

### CODE REVIEW

**Comment by journey2w** [comment]
@annie-samsung 
Please review English.

File: docs/application/native/guides/device/system.md [hunk 1]
```diff
@@ -159,6 +159,16 @@ The following table lists the consumer IR feature keys.
...
 | `http://tizen.org/feature/consumer_ir` | `bool` | The platform returns `true` for this key, if the device supports the Consumer Infrared (CIR) feature. |
 
+The following table lists the content feature keys.
+
+<a name="content"></a>
+**Table: Content feature keys**  
+
+| Key                                    | Type   | Description                              |
+|----------------------------------------|--------|------------------------------------------|
+| `http://tizen.org/feature/content.scanning.others` | `bool` | The platform returns true for this key, if the device supports the media scanning feature for OTHERS-type files which are not included in the media content types such as IMAGE, VIDEO, SOUND or MUSIC. |
```
Comment (surbhig12) [inline docs/application/native/guides/device/system.md]: Suggestion:

The platform returns true for this key, if the device supports the media scanning feature for `OTHERS` type files, which are not included in the media content-types such as image, video, sound, or music.

I have few questions:
1.	I have suggested using code tag for OTHERS as we are referring to the type of files. Please let me know if it’s okay.
2. Is the content-types IMAGE, VIDEO, SOUND or MUSIC in upper case as this is how the input is taken? If not, can we replace it with image, video, sound, or music?

Comment (haejeongkim) [reply docs/application/native/guides/device/system.md]: 1. OTHERS-type => OTHERS type is OK
2. input is in upper case like IMAGE, VIDEO.. So I think it's better not to change it.

Comment (surbhig12) [reply docs/application/native/guides/device/system.md]: @haejeongkim,

Thank you for clarifying my doubts.

Please rephrase the content to:

The platform returns true for this key, if the device supports the media scanning feature for OTHERS type files, which are not included in the media content-types such as IMAGE, VIDEO, SOUND, or MUSIC.

**Comment by journey2w** [comment]
@surbhig12 
If there is no issue, please approve this PR.

**Comment by mijin85cho** [comment]
Hello, @surbhig12 

When you check the update in device/system.md file, please check the location of added strings, too.

In the "System information keys" table, the items should be listed up in alphabetical order.
It follows alphabetical order, from top to bottom, and from left to right.
So the added "Content" should be located under "Consumer IR", above the "Contextual trigger",
and other items should be re-arranged.

Hello, @haejeongkim
Would you please re-arrange the items in "Featyre keys" section to the alphabetical order?
image

File: docs/application/native/guides/device/system.md [hunk 2]
```diff
@@ -159,6 +159,16 @@ The following table lists the consumer IR feature keys.
...
 | `http://tizen.org/feature/consumer_ir` | `bool` | The platform returns `true` for this key, if the device supports the Consumer Infrared (CIR) feature. |
 
+The following table lists the content feature keys.
+
+<a name="content"></a>
+**Table: Content feature keys**  
+
+| Key                                    | Type   | Description                              |
+|----------------------------------------|--------|------------------------------------------|
+| `http://tizen.org/feature/content.scanning.others` | `bool` | The platform returns true for this key, if the device supports the media scanning feature for OTHERS type files which are not included in the media content-types such as IMAGE, VIDEO, SOUND or MUSIC. |
```
Comment (mijin85cho) [inline docs/application/native/guides/device/system.md]: Please change "true" to \`true\` in here and next line.

The platform returns \`true\` for this key,...

Comment (haejeongkim) [reply docs/application/native/guides/device/system.md]: Done

**Comment by surbhig12** [comment]
@mijin85cho,

Thank you for letting me know. Going forward, I will check the location of added strings, too.

@haejeongkim,

As mentioned by @mijin85cho, in the Feature Keys table, please add  "Content" under "Consumer IR".

**Comment by mijin85cho** [comment]
Hello, @haejeongkim 
Thanks for the noti. I checked with  Ms. Kang.

Hello, @surbhig12 
As haejeongkim replied, this fix is confirmed by journey2w. So for this fix I ignore the mis-order.
As you confirmed other fix I consider this PR as approved by you. Thanks for the check.

**Comment by mijin80lee** [comment]
TD Sync Completed.

**Pull request merged**